query-primitives/src/main/scala/io/shiftleft/queryprimitives/steps/types/structure/MethodInst.scala

```diff
@@ -192,6 +192,6 @@ class MethodInst[Labels <: HList](override val raw: GremlinScala.Aux[nodes.Metho
     * Traverse to literals used in method.
     * */
   def literal: Literal[Labels] = {
-    method.literal
+    method.isLiteral
```

Comment:
This also does not seem to work because `isLiteral` is only a filter. We now have a `literal` and and `isLitertal` step on METHOD. Same goes for `isBlock` and `block`. I suggest we do not do this. I think defining the `isXXX` methods in AstNode does not go well with deriving Method from AstNode.

Reply:
Fixed.